application/models/Players.php

```diff
@@ -2,11 +2,6 @@
 
 defined('BASEPATH') OR exit('No direct script access allowed');
 
-/**
- * Data access wrapper for "orders" table.
- *
- * @author jim
- */
```

Comment:
Thanks...Though some comments should be kept.  I'll work on that later or you can.